src/traces/scattermapbox/defaults.js

```diff
@@ -28,6 +28,13 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
         return;
     }
 
+    coerce('cluster.maxZoom');
+    coerce('cluster.radius');
+    coerce('cluster.steps');
+    coerce('cluster.size');
+    coerce('cluster.color');
+    traceOut.cluster.enabled = traceIn.cluster ? true : false;
```

Comment:
This is not quite right at the moment e.g. it is not going to work if the `cluster` is declared via `template.
But after adjusting the new attributes, it should look like this.
```js
var clusterEnabled = coerce('cluster.enabled');
if(clusterEnabled) {
    coerce('cluster.maxzoom');
    coerce('cluster.radius');
    coerce('cluster.steps');
    coerce('cluster.size');
    coerce('cluster.color');
}
```